tests/test_controller.py

```diff
@@ -251,12 +251,12 @@ def _is_library(self, expected, actual, has_email=True):
                 actual_time = [actual_ts.year, actual_ts.month, actual_ts.day]
                 expected_time = [expected_ts.year, expected_ts.month, expected_ts.day]
                 eq_(actual_time, expected_time)
-            elif k == "contact_email":
+            elif "_email" in k:
```

Comment:
`k.endswith("_email")` will work the same way and avoid future false positives. This is in test code so it's not as important.